### PR TITLE
Add support for net9.0

### PR DESCRIPTION
**Pull request by AndyGerlicher**

### CODE REVIEW

**Review by erikmav — DISMISSED** [review]

File: Directory.Packages.props [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <MicrosoftBuildPackageVersion Condition="'$(TargetFramework)' == 'netstandard2.0'">16.9.0</MicrosoftBuildPackageVersion>
-    <MicrosoftBuildPackageVersion Condition="'$(TargetFramework)' == 'net46'">15.9.20</MicrosoftBuildPackageVersion>
-    <MicrosoftBuildPackageVersion Condition="'$(TargetFramework)' == 'net8.0'">17.7.2</MicrosoftBuildPackageVersion>
-    <MicrosoftBuildPackageVersion>17.8.3</MicrosoftBuildPackageVersion>
+
+    <!-- Please use caution adding a depenendcy to MSBuild packages. Use latest for testing and
```
Comment (dfederm) [inline Directory.Packages.props:5]: Nit: "depenendcy" => "dependency"

File: global.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "sdk": {
-    "version": "8.0.400",
-    "rollForward": "latestMinor"
+    "version": "9.0",
```
Comment (dfederm) [inline global.json:3]: I don't believe this value is valid

File: src/CentralPackageVersions/Microsoft.Build.CentralPackageVersions.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```
Comment (dfederm) [inline src/CentralPackageVersions/Microsoft.Build.CentralPackageVersions.csproj:3]: If we target `netstandard2.0`, we don't really need `net472`

File: src/NoTargets/Microsoft.Build.NoTargets.csproj [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```
Comment (dfederm) [inline src/NoTargets/Microsoft.Build.NoTargets.csproj:3]: Same re `netstandard2.0`

File: src/Traversal/Microsoft.Build.Traversal.csproj [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net472;netstandard2.0</TargetFrameworks>
```
Comment (dfederm) [inline src/Traversal/Microsoft.Build.Traversal.csproj:3]: Same

**Review by dfederm — APPROVED** [review]

**Pull request merged**